timeline_cards/styles/globals.css

```diff
@@ -29,4 +32,50 @@ li.div:not(:last-child) {
 
 p {
   padding-bottom: 10px;
-}
\ No newline at end of file
+}
+
+:root,
+div[data-theme="light"] {
+  --bg-color: white;
+  --font-color: black;
+  --social-icon-color: black;
+  --border-color: #eaeaea;
+  --card-color: white;
+  --card-text-color: black;
+  --close-card-color: gray;
+  --close-color: white;
+  --border-color-hover: #0070f3;
+  --footer-border-color: #eaeaea;
+  --theme-changer-icon-color: orange;
+  --theme-changer-inactive-icon: gray;
+}
+
+div[data-theme="pink"] {
+  --bg-color: pink;
+  --font-color: darkred;
+  --secondary-color: pink;
+  --social-icon-color: darkred;
+  --border-color: none;
+  --card-color: rgb(247, 160, 175);
+  --card-text-color: darkred;
+  --close-card-color: darkred;
+  --close-color: pink;
+  --border-color-hover: none;
+  --footer-border-color: darkred;
+  --theme-changer-icon-color: darkred;
+  --theme-changer-inactive-icon: rgb(185, 98, 112);
+}
+
+div[data-theme="dark"] {
+  --bg-color: #242424;
+  --font-color: lightgray;
+  --border-color: #2c2c2c;
+  --social-icon-color: lightgray;
+  --card-color: #2c2c2c;
+  --card-text-color: lightgray;
+  --close-color: #2c2c2c;
+  --border-color-hover: none;
+  --footer-border-color: lightgray;
+  --theme-changer-icon-color: yellow;
+  --theme-changer-inactive-icon: lightgray;
+}
```

Comment:
Here are I have set up 3 themes, a light (default) theme, dark theme and upon popular request a pink theme.